other/coding/src/main/java/org/example/coding/loadbalancer/model/Server.java

```diff
@@ -0,0 +1,33 @@
+package org.example.coding.loadbalancer.model;
+
+import java.util.Objects;
+
+public class Server {
+
+    private final String url;
+
+    public Server(String url) {
+        this.url = url;
```

Comment:
Do we need any validation for url? Is string good enough, or maybe we should go for java.net.URL?

Reply:
For simplicity I used just String, but for sure can change to URL